core-bundle/src/Resources/views/Backend/be_filesync_report.html.twig

```diff
@@ -0,0 +1,44 @@
+{% trans_default_domain 'contao_tl_files' %}
+
+{#  Header #}
+<div class="tl_message">
+    <p class="tl_confirm">{{ 'tl_files.syncComplete'|trans }}</p>
+</div>
+<div id="tl_buttons">
+    <a href="{{ referer }}" class="header_back" title="{{ 'MSC.backBTTitle'|trans({}, 'contao_default') }}" accesskey="b" onclick="Backend.getScrollOffset()">{{ 'MSC.backBT'|trans({}, 'contao_default') }}</a>
+</div>
+
+{# Report #}
+<div id="result-list">
+    {% set modified = change_set.itemsToUpdate|filter(updated => updated[constant('ATTR_HASH', change_set)] is defined) %}
+    {% set moved = change_set.itemsToUpdate|filter(updated => updated[constant('ATTR_PATH', change_set)] is defined) %}
+
+    <p>{{ 'tl_files.syncResult'|trans([change_set.itemsToCreate|length, modified|length, '-', moved|length, change_set.itemsToDelete|length], 'contao_tl_files')|raw }}</p>
+
+    {# Added resources #}
+    {% for added in change_set.itemsToCreate %}
+        <p class="tl_new">{{ 'tl_files.syncAdded'|trans([added[constant('ATTR_PATH', change_set)]]) }}</p>
+    {% endfor %}
+
+    {# Modified resources #}
+    {% for path, updated in modified %}
```

Comment:
It seems you are not using the `updated` variable?

Reply:
Would you prefer using `modified|keys`?

Reply:
Not sure. Isn‘t it `{% for path in modified %}` then?

Reply:
Ah, I see. Associative array. Never mind then. 👍